<?xml version="1.0" encoding="UTF-8"?><testrun name="SumTest" project="MathhEngine" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="MathEnginePackage.SumTest" time="0.042" displayname="SumTest" uniqueid="[engine:junit-jupiter]/[class:MathEnginePackage.SumTest]">
    <testcase name="testAddNumbers" classname="MathEnginePackage.SumTest" time="0.042" displayname="testAddNumbers()" uniqueid="[engine:junit-jupiter]/[class:MathEnginePackage.SumTest]/[method:testAddNumbers()]">
      <failure>
        <expected>15</expected>
        <actual>17</actual>
        org.opentest4j.AssertionFailedError: The addNumbers method should return the correct sum of two numbers. ==&gt; expected: &lt;15&gt; but was: &lt;17&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:563)&#13;
	at MathEnginePackage.SumTest.testAddNumbers(SumTest.java:16)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)&#13;
&#13;

      </failure>
    </testcase>
  </testsuite>
</testrun>
